java/configuration/src/main/java/sleeper/configuration/properties/instance/CompactionProperty.java

```diff
@@ -117,6 +117,14 @@ public interface CompactionProperty {
             .description("The maximum number of concurrent compaction tasks to run.")
             .defaultValue("300")
             .propertyGroup(InstancePropertyGroup.COMPACTION).build();
+    UserDefinedInstanceProperty COMPACTION_LAMBDA_CONCURRENCY_RESERVED = Index.propertyBuilder("sleeper.compaction.concurrency.reserved")
+            .description("The reserved concurrency for the compaction lambda.")
+            .defaultValue(null)
+            .propertyGroup(InstancePropertyGroup.COMPACTION).build();
+    UserDefinedInstanceProperty COMPACTION_LAMBDA_CONCURRENCY_MAXIMUM = Index.propertyBuilder("sleeper.compaction.concurrency.max")
+            .description("The maximum given concurrency allowed for the compaction lambda.")
```

Comment:
This is for the compaction job creation lambda, as opposed to the compaction task creation lambda. These properties can match the ones above starting with `sleeper.compaction.job.creation`.

Reply:
Addressed